javascript/ql/src/experimental/Security/CWE-829/pull_request_target.ql

```diff
@@ -0,0 +1,122 @@
+/**
+ * @name pull_request_target with explicit pull request checkout
```

Comment:
I got an very optional suggestion for the naming of the query. 

```suggestion
 * @name Checkout of untrusted code in trusted context
```

Also, could you change the filename to use CamelCase, and maybe name it something like `UntrustedCheckout.ql`.